modules/templates/helper.go

```diff
@@ -181,9 +181,21 @@ func NewFuncMap() template.FuncMap {
 
 		"FilenameIsImage": FilenameIsImage,
 		"TabSizeClass":    TabSizeClass,
+		"ShadowStorage":   shadowStorage,
```

Comment:
It's better to avoid bloating this template helper map.

It could be something like:

```
// ToShadowCopy ... this function is used by templates
func(storage *setting.Storage) ToShadowCopy() setting.Storage {
...
}
```

Reply:
Done.